app/template/rest.py

```diff
@@ -101,6 +101,8 @@ def create_template(service_id):
         char_limit = SMS_CHAR_COUNT_LIMIT if new_template.template_type == SMS_TYPE else EMAIL_CHAR_COUNT_LIMIT
         message = "Content has a character count greater than the limit of {}".format(char_limit)
         errors = {"content": [message]}
+        if new_template.template_type == EMAIL_TYPE:
+            current_app.logger.info({"error": "email_char_count_exceeded", "message": message, "service_id": service_id})
```

Comment:
This could be a `warn` level log. We could log it for SMS as well, if we want to build stats on these from logs.